api/apiServer.py

```diff
@@ -17,43 +17,43 @@ def root():
 def root():
     vagrantfile = "api/config/vb"
     v = vagrant.Vagrant(vagrantfile, quiet_stdout=False)
-    v.up() 
+    v.up() # For creating an vb instance based on vagrantfile
     return  {"message": "Local Virtualbox Instance Provisoned."}
     
 @app.get("/provision/vb/deleteInstance")
 def root():
     vagrantfile = "api/config/vb"
     v = vagrant.Vagrant(vagrantfile, quiet_stdout=False)
-    v.destroy() 
+    v.destroy() # For deleting an vb instance
     return  {"message": "Local Virtualbox Instance Destroyed."}
     
 @app.get("/control/vb/startInstance")
 def root():
     vagrantfile = "api/config/vb"
     v = vagrant.Vagrant(vagrantfile, quiet_stdout=False)
-    v.up() 
+    v.up() # For starting an vb instance
     return  {"message": "Local Virtualbox Instance Started"}
     
 @app.get("/control/vb/stopInstance")
 def root():
     vagrantfile = "api/config/vb"
     v = vagrant.Vagrant(vagrantfile, quiet_stdout=False)
-    v.halt() 
+    v.halt() # For halting an vb instance
     return  {"message": "Local Virtualbox Instance Halted."}
     
 @app.get("/resource/vb/statusInstance")
```

Comment:
It is status for all instances, right? If yes, let's change to "/resource/vb/statusInstanceAll".
We may need to have for the specific instance also in the future.